pkgs/development/python-modules/pytest-httpx/default.nix

```diff
@@ -13,6 +13,11 @@ buildPythonPackage rec {
 
   propagatedBuildInputs = [ httpx pytest ];
 
+  postPatch = ''
+    # https://github.com/Colin-b/pytest_httpx/issues/35
+    substituteInPlace setup.py --replace "httpx==0.16.*" "httpx>=0.16.*"
```

Comment:
You should rather update to https://github.com/Colin-b/pytest_httpx/releases/tag/v0.11.0.

Reply:
Oh, nice that they already providing an update.